tests/test_traits.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+
+import logging 
+
+from traitlets import HasTraits
+from traitlets.tests.test_traitlets import TraitTestBase
+from jupyter_telemetry.traits import HandlersList
+
+
+class HandlersListTrait(HasTraits):
+    
+    value = HandlersList(
+        None,
+        allow_none=True
+    )
+
+
+class TestHandlersList(TraitTestBase):
```

Comment:
It would be awesome if we can keep this be a py.test native function style test, to match other tests we have...

Reply:
Ok, I've refactored these tests to be more pytest-like.